prawcore/sessions.py

```diff
@@ -78,6 +78,33 @@ def should_retry_on_failure(self):
         return self._retries > 1
 
 
+class InfiniteRetryStrategy(RetryStrategy):
+    """A ``RetryStrategy`` that retries forever."""
+
+    def _sleep_seconds(self):
+        return self._base_delay + 2 * random.random()
+
+    def __init__(self, base_delay=2, max_delay=60):
+        """Initialize the strategy.
+
+        :param base_delay: Approximate time to sleep on the next retry.
+        :param max_delay: Maximum (approximate) time to sleep on any retry.
+
+        """
+        self._base_delay = max(base_delay, max_delay)
```

Comment:
This should be `self._base_delay = min(base_delay, max_delay)`

Reply:
```suggestion
        self._base_delay = min(base_delay, max_delay)
```

Reply:
It's amazing how consistently I screw that up.